src/components/ui/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import { TextareaHTMLAttributes } from 'react'
+import styles from './TextArea.module.css'
+
+interface ITextAreaProps extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+  error: boolean
+  errorMessage: string
+}
+
+export const TextArea = ({
```

Comment:
![123](https://github.com/user-attachments/assets/0fff483f-20f8-4565-876d-d2b76b3fe5a5)

Reply:
Hey @b3dyk , I took care of the scrollbar styling. Could you please take a quick look?